install/infra/terraform/gke/variables.tf

```diff
@@ -0,0 +1,81 @@
+variable "project" {
+  type        = string
+  description = "The project ID to create the cluster."
+}
+
+variable "kubeconfig" {
+  type        = string
+  description = "Path to write the kubeconfig output to"
+  default     = "./kubeconfig"
+}
+
+variable "region" {
+  type        = string
+  description = "The region to create the cluster."
+  default     = "europe-west1"
+}
+
+variable "zone" {
+  type        = string
+  description = "The zone to create the cluster."
+  default     = "b"
+}
+
+variable "kubernetes_version" {
+  type        = string
+  description = "Kubernetes version to be setup"
+  default     = "1.21.11-gke.1100"
+}
+
+variable "name" {
+  type        = string
+  description = "The name of the cluster."
+  default     = "sh-test"
+}
+
+variable "workspaces_machine_type" {
+  type        = string
+  description = "Type of the node compute engines for workspace nodepool."
+  default     = "n2-standard-8"
+}
+
+variable "services_machine_type" {
+  type        = string
+  description = "Type of the node compute engines for services nodepool."
+  default     = "n2-standard-4"
+}
+
+variable "min_count" {
+  type        = number
+  description = "Minimum number of nodes in the NodePool. Must be >=0 and <= max_node_count."
+  default     = 1
+}
+
+variable "max_count" {
+  type        = number
+  description = "Maximum number of nodes in the NodePool. Must be >= min_node_count."
+  default     = 50
+}
+
+variable "disk_size_gb" {
```

Comment:
Why make this configurable? I think the disk size can be decided statically based on what our services need.

Reply:
The assumption was that, if the user decides to use more in-cluster storage, it would mean that they need more storage. So keeping it configurable made sense at the time.